<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSymbol</class>
 <widget class="QWidget" name="SelectSymbol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>520</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLineEdit" name="editSearch">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFavorites">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Favorites</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableSymbols">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAll</sender>
   <signal>clicked()</signal>
   <receiver>SelectSymbol</receiver>
   <slot>OnShowAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFavorites</sender>
   <signal>clicked()</signal>
   <receiver>SelectSymbol</receiver>
   <slot>OnShowFavorites()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableSymbols</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>SelectSymbol</receiver>
   <slot>OnSymbolSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnShowAll()</slot>
  <slot>OnShowFavorites()</slot>
  <slot>OnSearchChange(QString)</slot>
  <slot>OnSymbolSelected(QModelIndex)</slot>
 </slots>
</ui>
